packages/onchainkit/src/internal/components/TextInput.tsx

```diff
@@ -63,11 +63,14 @@ export const TextInput = forwardRef(
 
     return (
       <input
+        aria-disabled={disabled}
+        aria-errormessage={errorMessage}
```

Comment:
issue: The value of `aria-errormessage` should be the ID of the element that renders the error message. Since we're not rendering the message, this doesn't really follow spec as is.